src/meetingsession/MeetingSessionStatus.ts

```diff
@@ -25,7 +25,6 @@ export default class MeetingSessionStatus {
       case MeetingSessionStatusCode.SignalingBadRequest:
       case MeetingSessionStatusCode.SignalingInternalServerError:
       case MeetingSessionStatusCode.SignalingRequestFailed:
-      case MeetingSessionStatusCode.StateMachineTransitionFailed:
```

Comment:
It's a bit of an involved effort to audit, especially since there is currently a blurry line between Chime application and SDK connections. I think these are fine for now :).

Reply:
Yeah we can do that but its not the goal of this PR which is just to remove plan B support. I just removed those because Kyu already marked them to be deprecated.

Reply:
we should start marking things as deprecated more :). Makes clean up much easier.